3rdParty/LightPcapNg/LightPcapNg/src/light_pcapng_ext.c

```diff
@@ -411,28 +411,37 @@ int light_get_next_packet(light_pcapng_t *pcapng, light_packet_header *packet_he
 		packet_header->interface_id = epb->interface_id;
 		packet_header->captured_length = epb->capture_packet_length;
 		packet_header->original_length = epb->original_capture_length;
-		uint64_t timestamp = epb->timestamp_high;
-		timestamp = timestamp << 32;
-		timestamp += epb->timestamp_low;
-		double timestamp_res = pcapng->file_info->timestamp_resolution[epb->interface_id];
-		uint64_t packet_secs = timestamp * timestamp_res;
-		if (packet_secs <= MAXIMUM_PACKET_SECONDS_VALUE)
+
+		if (epb->interface_id < pcapng->file_info->interface_block_count)
```

Comment:
could you add a comment between the block you made the change, so later we can easily upgrade the library.
e.g.
```
/// PCPP patch begin
....
/// PCPP patch end
```

Reply:
done.